supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -108,6 +108,37 @@
 
 *See also*: xref:ttl[TTL], xref:time-to-live-n[time to live]
 
+[discrete]
+[[tls]]
+==== image:images/yes.png[yes] TLS (noun)
+*Description*: TLS is an initialism for Transport Layer Security (TLS), and it is the successor to the Secure Sockets Layer (SSL) protocol.
+
+TLS is a cryptographic protocol that uses the Public Key Infrastructure (PKI) method to encrypt network traffic between two systems. PKI uses asymmetric encryption during a TLS handshake process to authenticate the connection between two systems.
+
+Use TLS when referring to protocols that exchange cryptographic keys and secure network connections between two systems. Check for the latest version of the TLS protocol and, if necessary, contact a subject matter expert (SME) to verify the TLS version to note in product documentation.
+
+Use SSL/TLS in high-level documentation entries, such as headings, to establish context with encryption protocols.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ssl[ssl], xref:ssl-tls[ssl-tls], xref:symmetric-encryption[symmetric encryption], xref:tls-handshake[TLS handshake], xref:trusted-certificate-authority[trusted Certificate Authority]
+
+
+[discrete]
+[[tls-handshake]]
+==== image:images/yes.png[yes] TLS handshake (noun)
+*Description*: The process of a client checking the validity of a certificate on a web server for authentication purposes.
+
+*Example*: A client requests a certificate from a web server. On receiving the certificate, the client checks that it trusts the certificate authority (CA) that issued the certificate. If the client trusts the CA, it generates a premaster secret and encrypts it by using the web server’s public key. The client sends the encrypted value to the web server. The web server decrypts the value by using its private key. Both client and web server calculate a shared session key by using the premaster secret and other values. Both client and web server then use the session key to encrypt any sent messages during the TLS session.
```

Comment:
Perhaps this example can be converted into a numbered list to facilitate reading.

Reply:
Hi @porueesq , your comment would make sense, but I do not see this style added to any other CCS SG entry. 

@redhat-documentation/ccs-style-council , WDYT?

Reply:
Thanks, @bergerhoffer . Good suggestion

:heavy_check_mark: